protos/core.proto

```diff
@@ -26,9 +28,8 @@ message CompileConfig {
   // Required.
   string project_dir = 1;
 
-  // Optional overrides for project-level settings.
-  string default_schema_override = 2;
-  string assertion_schema_override = 3;
+  // Compile time configuration options.
+  string schema_suffix = 2;
```

Comment:
i think we should either call this `schema_suffix_override` (if we plan to keep `schema_suffix` in `ProjectConfig`) or remove `schema_suffix` from `ProjectConfig`.

Reply:
Changed to `_override`